### PR TITLE
storage: Switch back from PreVote to CheckQuorum

### PR DESCRIPTION
**Pull request by bdarnell**
PreVote has bugs that show up under chaos testing.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/10395)

### CODE REVIEW

File: pkg/storage/replica.go [hunk 1]
```diff
@@ -2216,6 +2216,29 @@ func (r *Replica) tickRaftMuLocked() (bool, error) {
...
+		// By ticking the logical clock (incrementing an integer) we avoid this
+		// situation. If one of the followers does not quiesce it will call an
+		// election but the election will succeed. Note that while we expect such
+		// elections from quiesced followers to be extremely rare, it is very
+		// difficult to completely eliminate them so we want to minimize the
+		// disruption when they do occur.
+		//
+		// For more details, see #9372.
+		// TODO(bdarnell): remove this once we have fully switched to PreVote
+		if envutil.EnvOrDefaultBool("COCKROACH_TICK_QUIESCED", true) {
```
Comment (petermattis) [inline pkg/storage/replica.go]: Do you want to be parsing this env var on every tick? I think this should move into a `tickQuiesced` global variable.

**Review by petermattis — APPROVED** [review]
LGTM

**Comment by tamird** [comment]
LGTM. Can you cross-link to an upstream issue?

File: pkg/storage/replica.go (hunk 1 above)
Comment (tschottdorf) [inline pkg/storage/replica.go]: It probably doesn't matter and this call is cheap, but this is in a fairly common path. Perhaps `var tickQuiesced = envutil.EnvOrDefaultBool(...)` at beginning of file.

**Review by tbg — APPROVED** [review]

File: pkg/storage/replica.go (hunk 1 above)
Comment (tschottdorf) [inline pkg/storage/replica.go]: Also, why this env var and not just `!enablePreVoteRPC`?

Comment (bdarnell) [inline pkg/storage/replica.go]: Done

**Comment by bdarnell** [comment]
There's no issue to link to yet. We don't know enough about what's going on to file a useful issue.

File: pkg/storage/replica.go (hunk 1 above)
Comment (bdarnell) [inline pkg/storage/replica.go]: I want to be able to try with both enablePreVote and tickQuesced to see if the ticks fix the problems. (@petermattis originally suggested that the two changes go in as two separate PRs)

**Comment by a-robinson** [comment]
Merging at @bdarnell's request so that we can start verifying a new beta candidate
